assets/scss/00-base/_breakpoints.scss

```diff
@@ -8,19 +8,22 @@ $bp-l-ex: 1050px;
 $bp-xl: 1200px;
 
 
+// use of rem values for better responsive layouts 
+// and better accessibility at zoom levels
+
 // Global breakpoint media variables.
-$bp-x-small-min: "min-width: " + ($bp-xs + 1);
-$bp-x-small-max: "max-width:" + $bp-xs;
-$bp-small-min: "min-width: " + ($bp-s + 1);
-$bp-small-max: "max-width:" + $bp-s;
-$bp-medium-min: "min-width: " + ($bp-m + 1);
-$bp-medium-max: "max-width:" + $bp-m;
-$bp-large-min: "min-width: " + ($bp-l + 1);
-$bp-large-max: "max-width:" + $bp-l;
-$bp-large-extended: "max-width:" + $bp-l-ex;
-$bp-large-extended-min: "min-width: " + ($bp-l-ex + 1);
-$bp-x-large-min: "min-width: " + ($bp-xl + 1);
-$bp-x-large-max: "max-width:" + $bp-xl;
+$bp-x-small-min: "min-width: 30.06rem"; // 481px

```

Comment:
The issue being addressed by the ticket involved overlapping sections when the screen was zoomed in. The example video shows the issue on a howto page with a sticky sidebar menu. 
Zoom levels don't change the screens pixel width, so zooming in can lead to overlapping of positioned elements.
By changing the values from pixels to rem, when you zoom in the media queries kick in on zoom as well as screen size. 
So, using the howto page as an example again, when you zoom into the page, the mobile styles kick back in, rather than the elements overlapping.

Reply:
Ok.  That makes sense.  Thank you for the answer.